rxp-hpp-android/src/main/java/com/realexpayments/hpp/ApiAdapter.java

```diff
@@ -19,13 +19,22 @@
     public static final String RETROFIT_TAG = "HPPRetrofit";
 
     public static IHPPServerAPI getAdapter(String endpoint, Map<String, String> headers) {
-        RestAdapter restAdapter = new RestAdapter.Builder()
-                .setEndpoint(endpoint)
-                .setLogLevel(RestAdapter.LogLevel.FULL)
-                .setLog(new AndroidLog(RETROFIT_TAG))
-                .setConverter(new GsonConverter(getGson()))
-                .setRequestInterceptor(getRequestInterceptor(headers))
-                .build();
+        RestAdapter restAdapter;
+        if (BuildConfig.DEBUG) {
```

Comment:
This would be from integrators' build configs and not only ours, correct?

Reply:
@eduardo76609 Are you able to confirm if the integrator provides the build config?

Reply:
Hello Shane,
when the integrator builds his app, this class is generated immediately, @pablodamiancoca had the same problem and when he compiled the app the class was resolved